backend/internal/utils/user/create_account_test.go

```diff
@@ -0,0 +1,32 @@
+package user
+
+import (
+	"testing"
+)
+
```

Comment:
Nice work on this test! Have you done go before or did you learn this type of testing yourself?

Reply:
I've never done any backend stuff or any go/unit testing, just a lot of learning this last week, I thought it was really cool that go has some built in testing though.